go.mod

```diff
@@ -121,9 +121,9 @@ require (
 	github.com/cyphar/filepath-securejoin v0.2.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/daviddengcn/go-colortext v1.0.0 // indirect
-	github.com/docker/cli v20.10.21+incompatible // indirect
+	github.com/docker/cli v23.0.1+incompatible // indirect
```

Comment:
Not sure about that Docker dependency bump. Maybe there are other libs that we can use that don't depend on Docker? There's e.g. [github.com/google/go-containerregistry](https://github.com/google/go-containerregistry).

Reply:
> Maybe there are other libs that we can use that don't depend on Docker?
I agree. Having docker as a dependency is always problematic... I think that if you found a suitable alternative, we should replace it.

OTOH oras-go doesn't seem necessary anymore:
```
➜  k0s git:(main) go mod why oras.land/oras-go v1.2.2
# oras.land/oras-go
(main module does not need package oras.land/oras-go)

# v1.2.2
(main module does not need package v1.2.2)
```

I will file a PR to remove the dependency.

Reply:
OK I tried to remove oras-go and turns out it is indeed necessary for helm.

Reply:
NB: github.com/google/go-containerregistry also depends on Docker. All those modules seem to rely on the Docker auth parts there.